deploy/kubernetes/Makefile

```diff
@@ -30,7 +30,7 @@ $(ALL):
 ifdef WITHOUTCONFIG
 	@echo "Generating $@-kubernetes-without-configmap.yaml"
 	@rm -f $@-kubernetes-without-configmap.yaml
-	@for f in $(shell ls $@/*.yaml | grep -v daemonset-configmap); do \
+	@for f in $(shell ls $@/*.yaml | grep -v daemonset-configmap | grep -v statefulset-configmap); do \
```

Comment:
is there any reason to not use grep -v "-configmap"'?

Reply:
Apparently not :) 
I changed it in [simplifying grep](https://github.com/elastic/elastic-agent/pull/3396/commits/6bfbd889b53e4b651b0ae32e6aac3cdeef01ad03)